src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,115 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.UserRegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class RegistrationServiceImplTest {
+    private static final int MIN_AGE = 18;
+    private static final String VALID_LOGIN = "abcdef";
+    private static final String VALID_PASSWORD = "123456";
+    private static final String LOGIN_LESS_THAN_MIN = "abcde";
+    private static final String PASSWORD_LESS_THAN_MIN = "12345";
+    private static final int AGE_LESS_THAN_MIN = 15;
+    private static RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
+    private final User user = new User(VALID_LOGIN, VALID_PASSWORD, MIN_AGE);
```

Comment:
instantiate user and use it where you're testing it. 

no need to make all strings\primitive constants, 
create constant only on demand